lib/phlex/test_helpers.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Phlex
+	module TestHelpers
```

Comment:
We should either move this to `Phlex::Rails::TestHelpers` or make these not depend directly on Rails. For example, we could use `call` directly if `ActionView::TestCase::TestController` isn't defined.